cli/cmd/get.go

```diff
@@ -16,7 +16,7 @@ var getCmd = &cobra.Command{
 
 Valid resource types include:
  * pods (aka pod, po)`,
-	RunE: func(cmd *cobra.Command, args []string) error {
+	Run: exitSilentlyOnError(func(cmd *cobra.Command, args []string) error {
```

Comment:
I don't think this is the right fix for this issue unfortunately. If this method returns errors (such as those on lines 39 and 44 in this file), then we should be surfacing those errors instead of exiting silently.

By way of example, on conduit master, if I run `go run cli/main.go get foo`, then I get:

```
$ go run cli/main.go get foo
Error: invalid resource type
Usage:
  conduit get [flags] RESOURCE

Flags:
      --api-addr string     Override kubeconfig and communicate directly with the control plane at host:port (mostly for testing)
  -h, --help                help for get
      --kubeconfig string   Path to the kubeconfig file to use for CLI requests (default "/Users/kl/.kube/config")

Global Flags:
  -n, --conduit-namespace string   namespace in which Conduit is installed (default "conduit")

invalid resource type
exit status 1
```

But on your branch, I get no output:

```
$ go run cli/main.go get foo
exit status 2
```

I think the intent of exitSilentlyOnError is to suppress the help message, but that's really only appropriate for the version command, where we're not validating user input.

I think a better approach to fixing the stacktrace would be to try to figure out which code is panicking, and then add code to recover from the panic before the stacktrace is printed.

Reply:
Ahh ok, I can revisit that, I was under the impression the `exitSilentlyOnError` was only silently erroring out on OS level errors like the ones you run into when using kubectl with the cli. Thanks for that